lib/tasks/locale.rake

```diff
@@ -171,6 +171,20 @@ namespace :locale do
 
   desc "Update all ManageIQ gettext catalogs and merge them into one"
   task "update_all" do
+    def remove_line_numbers(path)
+      puts "Removing line numbers from #{path}"
+
+      # Remove line numbers from source references
+      path.write(
+        path.readlines.map { |l| l.gsub(/^(#.+):[0-9]+\n$/, "\\1\n") }.join
+      )
+
+      # Fix duplicate subsequent lines where there were multiple references in the same file
+      path.write(
+        (path.readlines << nil).each_cons(2).map { |l1, l2| l1 == l2 ? nil : l1 }.compact.join
```

Comment:
Is this looking at the msgid/msgstr lines too?  Should we only look at consecutive duplicate `#` comment lines or more specifically ones with paths?

Reply:
Oh yes good point. This only look at _any_ duplicated subsequent lines.  I'd be surprised if any non-comment were identical, but I should probably be more tactical here.

Reply:
@jrafanie Updated.

Reply:
Also I took your word "consecutive" - I knew there was a better word 😆

Reply:
haha, naming is hard 🤣